scripts/build_test.py

```diff
@@ -201,43 +202,43 @@ def test_compare_file_count(self):
         # Reset EMPTY_DIRECTORY to clean state.
         build.safe_delete_directory_tree(EMPTY_DIR)
 
-    def test_verify_filepath_hash(self):
+    def test_verify_filepath_hash(self) -> None:
         """Test _verify_filepath_hash raises exception:
             1) When there is an empty hash dict.
             2) When a filename is expected to contain hash but does not.
             3) When there is a hash in filename that cannot be found in
                 hash dict.
         """
         # Final filepath example: base.240933e7564bd72a4dde42ee23260c5f.html.
-        file_hashes = {}
+        file_hashes: Dict[str, str] = {}
         base_filename = 'base.html'
-        with self.assertRaisesRegex(ValueError, 'Hash dict is empty'):
+        with self.assertRaisesRegex(ValueError, 'Hash dict is empty'):  # type: ignore[no-untyped-call]
             build._verify_filepath_hash(base_filename, file_hashes)  # pylint: disable=protected-access
 
         # Generate a random hash dict for base.html.
-        file_hashes = {base_filename: random.getrandbits(128)}
-        with self.assertRaisesRegex(
+        file_hashes = {base_filename: 'abcdfabcdfabcdf1ABCDFABCDFABCDF1'}
+        with self.assertRaisesRegex(  # type: ignore[no-untyped-call]
             ValueError, '%s is expected to contain MD5 hash' % base_filename):
             build._verify_filepath_hash(base_filename, file_hashes)  # pylint: disable=protected-access
 
         base_without_hash_filename = 'base_without_hash.html'
-        self.assertIsNone(build._verify_filepath_hash(  # pylint: disable=protected-access
+        self.assertIsNone(build._verify_filepath_hash(   # type: ignore[func-returns-value]  # pylint: disable=protected-access
```

Comment:
Why is this ignore needed?

Reply:
Hey, previously I asked you for this and to give you more context, I added this ignore and created this PR.. still, I am trying to figure out the reason behind this error.
### scripts/build_test.py:225: error: "_verify_filepath_hash" does not return a value  [func-returns-value]
from this error log, it seems like `_verify_filepath_hash` is not returning a value but by adding print statement I found it returning `None` as below.
![ss_7](https://user-images.githubusercontent.com/89520981/158049201-b6ec8b5d-275c-42ec-80bd-2b56a6b7abbf.png)
PTAL and give your suggestion. Thanks

Reply:
Ah I see. The problem is that there is no `return` in `_verify_filepath_hash`, thus MyPy does not expect that we would be checking it here. I think we should just remove this assert.

Reply:
hey, I removed the assert but as you can see [here](https://github.com/oppia/oppia/runs/5581993987?check_suite_focus=true) it left [this](https://github.com/oppia/oppia/blob/96a1c7a2b8310d12794ac55ba51e1d5603df3c39/scripts/build.py#L1169) line uncovered. what you would suggest here.

Reply:
Only remove the assert, do not remove the actual function call.

Reply:
updated